docs/api/topics/latecustomization.rst

```diff
@@ -0,0 +1,68 @@
+.. _latecustomization:
+.. versionadded:: 2
+
+====================
+ Late Customization
+====================
+
+Late customization allows loading of apps during the first-time experience on
+devices.
+
+.. http:get:: /api/v2/latecustomization/?carrier=(str:carrier)&region=(str:region)
+
+  Returns a list of late customization items for this carrier/region.
+
+  **Response**
+
+  :param app: A webapp ID.
+  :type app: int
+  :param region: A region ID.
+  :type region: int
```

Comment:
Same as below, the API seems to be returning slugs, not IDs (a good thing IMHO, but docs should be updated)